src/main/java/io/knotx/junit5/util/FreePortFinder.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2019 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.knotx.junit5.util;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+import org.apache.commons.lang3.RandomUtils;
+
+public final class FreePortFinder {
+
+  /** Util class */
+  private FreePortFinder() {}
+
+  /** Roll a port number and ensure it's available */
```

Comment:
Compilation warning:
> Task :knotx-junit5:javadoc
/Users/tomasz.michalak/knotx/knotx-junit5/src/main/java/io/knotx/junit5/util/FreePortFinder.java:28: warning: no @return
  public static int findFreeLocalPort() {

Reply:
I see you resolved it yourself, thanks!

Google style guidelines (which I use in this module) permit no return Javadoc, I must have missed the warning.